index.cjs

```diff
@@ -728,8 +728,35 @@ const config = {
   ],
 };
 
+// This class extends the built-in Error class and provides additional
+// functionality for error handling and message formatting.
+class AddLineBreaksForEslintOutputInVscodeError extends Error {
+  /**
+   * @param {string} message
+   */
+  constructor(message) {
+    super();
+    this._message = message;
+    this._firstErrorReplaced = false;
```

Comment:
The purpose of `this._firstErrorReplaced = false;` is to keep track of whether the replacement has been done for the first occurrence of "Error:". It is necessary to avoid replacements for other occurrences. Without this change, the error message would look like this:
<img width="1014" alt="Screenshot 2023-05-19 at 12 49 44" src="https://github.com/upleveled/eslint-config-upleveled/assets/80746311/57947f83-a227-46a2-a665-2646cf2780aa">
I tried including regular expressions to identify only the first instance of "Error:", but this didn't give me the correct results.